src/bucket/Bucket.cpp

```diff
@@ -91,6 +91,7 @@ Bucket::apply(Application& app) const
     BucketApplicator applicator(
         app, app.getConfig().LEDGER_PROTOCOL_VERSION,
         0 /*set to 0 so we always load from the parent to check state*/,
+        0 /*set to non-top level to we don't treat live entries as init*/,
```

Comment:
Nitpick: typo "to we" -> "so we"

But also, does this comment say the right thing? I think you mean that you don't want to set it to the bottom level.

Reply:
Fixed

Reply:
Should be good now.